README.md

```diff
@@ -1 +1,64 @@
-# Beyond-07-team-4
\ No newline at end of file
+# Job Seeker
+
+**Job Seeker** is a web platform, built for people to post and find jobs.
+It might serve students, freelancers or organzations.
+
+## How To Run The Project
+
+> NOTE: Depending on your OS, the commands in this README should be run with the following shells:
+> - *Windows:* on **cmd** or **Powershell**.
+> - *Linux:* on **terminal**.
+
+### Prerequisites
+
```

Comment:
Add Git as prerequisite .

Reply:
Is it a prerequisite though?
It is mentioned in the file that people who don't have Git installed may download the project manually by pressing the green "Code" button.

Reply:
> Is it a prerequisite though? It is mentioned in the file that people who don't have Git installed may download the project manually by pressing the green "Code" button.

@taljacob2 since there is no package to deploy/install the project
Only option to consume through git,

What audience will download through curl/download?
Let me suggest a prereq Note above git client:
Advised to use git to download this project.

MHO, no one will download it without git :),

Reply:
Alright.
- I have added Git as a *prerequisite* for this project. 
- I removed the section that explains how to download the project *without* Git.